ansys/dpf/core/meshed_region.py

```diff
@@ -133,9 +133,7 @@ def elements(self):
         DPF Elements object with 8 elements
 
         """
-        if self._elements is None:
-            self._elements = Elements(self)
-        return self._elements
```

Comment:
@cbellot000 That was a quick fix for a problem with vtk grids.

Reply:
@PProfizi what do you mean? I didn't take the fix from anywhere, I'm fixing a circular ref here

Reply:
@cbellot000 ah yes sorry I confused this with another part of code.